src/components/MeasurementOverlay/MeasurementOverlay.js

```diff
@@ -146,7 +147,7 @@ class MeasurementOverlay extends React.PureComponent {
     if (key === 'perimeterMeasurement' || key === 'areaMeasurement') {
       // for polyline and polygon, there's no useful information we can show if it has no vertices or only one vertex.
       showInfo = annotation.getPath().length > 1;
-    } else if (key === 'distanceMeasurement') {
+    } else if (key === 'distanceMeasurement' || key === 'ellipseMeasurement') {
       showInfo = true;
```

Comment:
I noticed an edge case where if I just move my mouse a tiny bit when creating the ellipse measurement annotation, I won't see the annotation on the canvas, but the overlay will show up with area being 0 and the overlay won't go away. I feel like we need to do some extra check here(for example, if the circle is small) instead of just checking the key.